exec/cnex/lib/os_win32.c

```diff
@@ -0,0 +1,154 @@
+#include "os.h"
+
+#include <iso646.h>
+#include <windows.h>
+
+#include "assert.h"
+#include "cell.h"
+#include "enums.h"
+#include "exec.h"
+#include "object.h"
+#include "stack.h"
+
+
+
+static Object *check_process(TExecutor *exec, Object *pp)
+{
+    if (pp == NULL || pp->ptr == INVALID_HANDLE_VALUE) {
+        exec->rtl_raise(exec, "OsException.InvalidProcess", "", BID_ZERO);
+        return NULL;
+    }
+    return pp;
+}
+
+void object_releaseProcessObject(Object *o)
+{
+    if (o != NULL) {
+        assert(o->refcount > 0);
+        o->refcount--;
+
+        if (o->refcount <= 0) {
+            if (o->ptr != INVALID_HANDLE_VALUE) {
+                CloseHandle(o->ptr);
+            }
+            free(o);
+        }
+    }
+}
+
+Cell *object_processObjectToString(Object *self)
+{
+    Cell *r = cell_fromCString("<PROCESS ");
+    char s[32];
+    snprintf(s, 32, "%llu>", (int64_t)self->ptr);
```

Comment:
Same comment about string formatting as above.

Reply:
Done, and still fits within the 32 chars, even a full 64 bit number.